### PR TITLE
8-patch a review, ready for code review

### PR DESCRIPTION
**Pull request by LKFULLER1**

### CODE REVIEW

File: __tests__/app.test.js [hunk 1]
```diff
@@ -238,4 +237,29 @@ describe('7. POST /api/reviews/:review_id/comments', () => {
...
+    const updateVotes = {
+      inc_votes: 5
+    };
+    return request(app)
+      .patch(`/api/reviews/${REVIEW_ID}`)
+      .send(updateVotes)
+      .expect(202)
+      .then(({ body }) => {
+        expect(body.review).toMatchObject({
+          review_id: 3,
```
Comment (MKD01) [inline __tests__/app.test.js]: Im happy for you to use variables for REVIEW_ID, but this 3 should also but that variable as well just for consistency.

File: __tests__/app.test.js [hunk 2]
```diff
@@ -261,5 +261,19 @@ describe('8. PATCH /api/reviews/:review_id', () => {
...
+      inc_votes: 5
+    };
+    return request(app)
+      .patch(`/api/reviews/${REVIEW_ID}`)
+      .send(updateVotes)
+      .expect(404)
+      .then(({ body }) => {
+        expect(body.msg).toBe('review not found!');
+      });
+  });
```
Comment (MKD01) [inline __tests__/app.test.js:272]: Good error handling but could be more for this? what if we were ti give an invalid id? or if the body given does not is not correct? Please do add tests for these as well.

File: controllers/games.js [hunk 3]
```diff
@@ -41,10 +41,7 @@ exports.postCommentByReviewId = (req, res, next) => {
 }
 
 exports.patchReview = (req, res, next) => {
-  //console.log(req.body)
-  //console.log(req.params.review_id)
   updateReview(req.body, req.params.review_id).then((review) => {
-    //console.log(review)
```
Comment (MKD01) [inline controllers/games.js]: good catch 👍

File: models/games.js [hunk 4]
```diff
@@ -60,5 +60,17 @@ exports.insertCommentByReviewId = (newComment) => {
     })
 };
 
+exports.updateReview = (updateVotes, review_id) => {
+    return this.selectReviewById(review_id).then(() => {
```
Comment (MKD01) [inline models/games.js]: This check doesnt need to be here, If you give this query an invalid review_id you can handle that error after based on the response.

**Review by MKD01 — CHANGES_REQUESTED** [review]
Please do make those changes and then send another PR to reflect this

File: models/games.js (hunk 4 above)
Comment (LKFULLER1) [reply models/games.js]: unsure on what you mean by this. This is just me reusing a function to test if the review_id exists, before continuing, as Liam was doing in the lecture yesterday

Comment (MKD01) [reply models/games.js]: What was showed in the lecture is correct, but for this endpoint using a helper function is unnecessary. Simply checking the result and throwing an error based off of that would be more than enough.

Comment (LKFULLER1) [reply models/games.js]: Hi, when trying to remove this check, there is no error as the sql will just update nothing. Are you saying to also remove this test?

**Comment by MKD01** [comment]
Happy for this to be merged after those changes have been made 👍

**Pull request merged**